scripts/dockerenv.sh

```diff
@@ -79,6 +79,7 @@ dockerdev () {
            --name="$CONTAINER_NAME" \
            -v "$repo_path":"$MOUNT_DIR" \
            --cap-add SYS_PTRACE \
+           -p 15423:15423 \
```

Comment:
please add a comment, it's not obvious what this is for when one does not know

Reply:
Git blame ;), jk. I'll add a comment

Reply:
done, force pushed